.github/workflows/longevity_test.yml

```diff
@@ -194,6 +191,48 @@ jobs:
 
 ##########################################################################################################################################
 
+# Backup Exchange Deletion test
+      - name: Backup Delete exchange test
+        id: delete-exchange-test
+        env:
+          SERVICE: "exchange"
+          DELETION_DAYS: 10
+        run: |
+          set -euo pipefail
+          echo -e "\nDelete Backup Exchange test\n" >> ${CORSO_LOG_FILE}
+          CORSO_LOG_FILE=${{ env.CORSO_LOG_DIR }}/backup_exchange.log
```

Comment:
will this env var assignment be used in the `./longevity-test` command? I'm a little rusty on bash, but I thought either it would need to be like `export CORSO_LOG_FILE=...` (to end up in the environment from that point forward) or be on the same line as the command being run (to be used as part of the environment only for the command being run)